dockerfile_parse/parser.py

```diff
@@ -100,6 +107,15 @@ def __init__(self, path=None, cache_content=False, env_replace=True, parent_env=
         else:
             self.parent_env = {}
 
+    @contextmanager
+    def _open_dockerfile(self, mode):
+        if self.fileobj is not None:
+            self.fileobj.seek(0)
```

Comment:
Some file-like objects will not support seek() and will raise `IOError`. What should we do in that situation?